---
# Documentation: https://wowchemy.com/docs/managing-content/

title: "Integrative analysis of DNA methylation and gene expression in Papillary Renal Cell Carcinoma"
authors: [**"Noor Pratap Singh"**, "Vinod PK"]
date: 2020-01-09T17:25:46-05:00
doi: "https://doi.org/10.1007/s00438-020-01664-y"

# Schedule page publish date (NOT publication's date).
publishDate: 2021-08-25T14:25:35-04:00

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "Molecular Genetics and Genomics"
publication_short: "MGAG"

abstract: "DNA methylation is an epigenetic mark that is significantly altered in cancer. Interpreting the functional consequences of DNA methylation requires integration of multiple forms of data. The recent advancement in the next generation sequencing can help to decode this relationship and in biomarker discovery. In this study, we investigated the methylation patterns of Papillary renal cell carcinoma (PRCC) and its relationship with the gene expression using The Cancer Genome Atlas (TCGA) multi-omics data. We found that the promoter and body of tumor suppressor genes, microRNAs and gene clusters and families including cadherins, protocadherins, claudins and collagens are hypermethylated in PRCC. Hypomethylated genes in PRCC are associated with the immune function. The gene expression of several novel candidate genes including interleukin receptor IL17RE and immune checkpoint genes HHLA2, SIRPA and HAVCR2 shows significant correlation with the DNA methylation. We also developed machine learning models using features extracted from single and multi-omics data to distinguish early and late stages of PRCC. A comparative study of different feature selection algorithms, predictive models, data integration techniques and representations of methylation data was performed. The Group Lasso (GL) model using both the gene expression and DNA methylation features shows the overall best performance in distinguishing tumor stages. In summary, our study identifies PRCC driver genes and proposes predictive models based on both DNA methylation and gene expression. These results on PRCC will aid in targeted experiments and provide a strategy to improve the classification accuracy of tumor stages."

# Summary. An optional shortened abstract.
summary: ""

tags: ["Renal cell carcinoma", "Multi-omics", "Epigenetic regulation", "RNASeq", "Data integration", "Multiple kernel learning", "Tumor stage prediction"]
categories: []
featured: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_pdf: https://link.springer.com/article/10.1007%2Fs00438-020-01664-y
url_code: https://github.com/NPSDC/IGAMS
url_dataset:
url_poster:
url_project:
url_slides:
url_source:
url_video:

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---
